uast/role_string.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by "stringer -type=Role"; DO NOT EDIT.
+// Code generated by "stringer -type=Role"; DO NOT EDIT
```

Comment:
I've not the mininal idea of how that could have changed but it will be rewritten on the next go generate anyway